packages/shared/src/components/modals/FeedFiltersModal/FeedFiltersModal.tsx

```diff
@@ -1,19 +1,23 @@
 import React, { ReactElement } from 'react';
 import classNames from 'classnames';
-import { ModalProps } from './StyledModal';
-import { ResponsiveModal } from './ResponsiveModal';
-import styles from './AccountDetailsModal.module.css';
-import TagsFilter from '../filters/TagsFilter';
-import CreateFeedFilterButton from '../CreateFeedFilterButton';
-import PlusIcon from '../../../icons/plus.svg';
-import XIcon from '../../../icons/x.svg';
-import { Button } from '../buttons/Button';
+import { ModalProps } from '../StyledModal';
+import { ResponsiveModal } from '../ResponsiveModal';
+import TagsFilter from '../../filters/TagsFilter';
+import CreateFeedFilterButton from '../../CreateFeedFilterButton';
+import PlusIcon from '../../../../icons/plus.svg';
+import XIcon from '../../../../icons/x.svg';
+import { Button } from '../../buttons/Button';
 
 type TypeProps = {
   feedFilterModalType: string;
+  showIntroModal: boolean;
+  feedFilterOnboardingModalType: string;
+  actionToOpenFeedFilters: () => unknown;
+  onIntroClose: () => unknown;
+  onCloseFeedFilterModal: () => unknown;
```

Comment:
This property shouldn't actually be necessary anymore since the type you combine this with already has the property that handles the closing of the modal. I suggest using that one than creating an additional property.
The property I mean to be utilized is the `onRequestClose` rather than this new one.
If you want the property name to be more intuitive, you can put an alias to the said property on destructuring.
```suggestion
```